static/src/stylesheets/module/commercial/glabs/_hosted.scss

```diff
@@ -155,7 +155,7 @@ $hosted-stripe: 28px;
         padding-top: 21px;
     }
 
-    &.vjs .vjs-play-control {
+    &.vjs .vjs-button.vjs-play-control {
```

Comment:
I was overwriting the styles of the video buttons (because at some point there will be video in the thrasher). Originally I was doing that only on the hosted page (http://m.code.dev-theguardian.com/commercial/guardian-hosted) but it seems that on fronts (where thrasher will be located) there is a different order of css files.